pkg/controllers/metrics/node/counter.go

```diff
@@ -0,0 +1,243 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package node
+
+import (
+	"strings"
+
+	karpenterv1 "github.com/awslabs/karpenter/pkg/apis/provisioning/v1alpha4"
```

Comment:
Could you rename this `provisioning`? (Or just use `v1alpha4` which the rest of the codebase does)?